src/components/input_fields/input_field_wrapper.tsx

```diff
@@ -76,99 +80,117 @@ export class InputFieldWrapper extends React.Component<InputFieldWrapperProps, {
+            );
         }
         return null;
     }
 
     private _renderValidation(): JSX.Element {
-        const {validationError, labelLayoutWidth, labelWidthBreakpoint, required} = this.props;
+        const {
+            validationError, labelLayoutWidth, labelWidthBreakpoint, required,
+        } = this.props;
 
         if (this._showValidationError()) {
-            return <InputFieldValidation
-                validationError={validationError}
-                labelLayoutWidth={labelLayoutWidth}
-                labelWidthBreakpoint={labelWidthBreakpoint}
-                required={required}
-            />;
+            return (
+                <InputFieldValidation
+                    validationError={validationError}
+                    labelLayoutWidth={labelLayoutWidth}
+                    labelWidthBreakpoint={labelWidthBreakpoint}
+                    required={required}
+                />
+            );
         }
         return null;
     }
 
-
-    render(): JSX.Element {
-        const {className, staticField} = this.props;
-
-        const inputClasses = classNames(`input-field layout`, className, {'hide-input': staticField});
-        return (
-            <div className={inputClasses}>
-                {this._renderLabel()}
-                {this._renderInput()}
-                {this._renderInputChildren()}
-                {this._renderValidation()}
-            </div>
-        );
-    };
-
     private _renderInputChildren(): JSX.Element {
         const { inputChildren } = this.props;
         if (inputChildren) {
-            return <div className="layout__item u-fit input-children">
-                {inputChildren}
-            </div>
+            return (
+                <div className="layout__item u-fit input-children">
+                    {inputChildren}
+                </div>
+            );
         }
         return null;
     }
 
     private _renderInput() {
-        const {labelLayoutWidth, centerInputField, staticField, className, children} = this.props;
+        const {
+            labelLayoutWidth, centerInputField, staticField, className, children,
+        } = this.props;
 
         const inputLayoutWidth = staticField || labelLayoutWidth === 'fill' ? 'fit' : 'fill';
         const inputLayoutClasses = classNames(
             `layout__item u-${inputLayoutWidth} layout__item--middle`,
             className,
-            {'text-center': centerInputField}
+            { 'text-center': centerInputField },
         );
 
-        return <div className={inputLayoutClasses}>
-            {!staticField &&
-            <div className="input-container layout layout--gutter-xs">
-                <div className="input-wrapper-children layout__item u-fill">
-                    {children}
-                </div>
-                {this._renderPostInputElement()}
+        return (
+            <div className={inputLayoutClasses}>
+                {!staticField
+                    && (
```

Comment:
why are there extra parentheses?

Reply:
Airbnb rule, but I have removed the line break